NuGet.config

```diff
@@ -13,6 +13,7 @@
     <add key="dotnet5-roslyn" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet5/nuget/v3/index.json" />
     <add key="mlnet-daily" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/MachineLearning/nuget/v3/index.json" />
     <add key="mlnet-assets" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/machinelearning-assets/nuget/v3/index.json" />
+    <add key="nuget" value="https://api.nuget.org/v3/index.json" />
```

Comment:
I'll have to defer to @ViktorHofer or @mmitche for certain, but I think we're supposed to exclusively use our dnceng feeds.

Reply:
Yep, this should be replaced by the "dotnet-public" feed if it's not already there.

Reply:
Yeah, this was just so I could get it up for review faster. The appropriate package has been mirrored and nuget is removed as a feed.